spaceship.zsh

```diff
@@ -405,39 +405,89 @@ spaceship_dir() {
 }
 
 # GIT BRANCH
-# Show current git brunch using git_current_status from Oh-My-Zsh
+# Show current git branch
 spaceship_git_branch() {
   [[ $SPACEHIP_GIT_BRANCH_SHOW == false ]] && return
 
   _is_git || return
 
+  local ref
+  ref=$(command git symbolic-ref --quiet HEAD 2> /dev/null)
+  local ret=$?
+  if [[ $ret != 0 ]]; then
+    [[ $ret == 128 ]] && return
+    ref=$(command git rev-parse --short HEAD 2> /dev/null) || return
+  fi
+
+  local git_current_branch=${ref#refs/heads/}
+
   _prompt_section \
     "$SPACESHIP_GIT_BRANCH_COLOR" \
     "$SPACESHIP_GIT_BRANCH_PREFIX$(git_current_branch)$SPACESHIP_GIT_BRANCH_SUFFIX"
 }
 
 # GIT STATUS
-# Check if current dir is a git repo, set up ZSH_THEME_* variables
-# and show git status using git_prompt_status from Oh-My-Zsh
-# Reference:
-#   https://github.com/robbyrussell/oh-my-zsh/blob/master/lib/git.zsh
+# Show git status
```

Comment:
Let's add a reference to this PR and small explanation why we don't use OMZ functions.